pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "MergePythonClient"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
@rmkonnur is the right updated version `0.1.1`

Reply:
nope should be 0.1.2